src/persist-client/src/read.rs

```diff
@@ -712,8 +713,11 @@ where
         // NB: min_elapsed is intentionally smaller than the one in
         // maybe_heartbeat_reader (this is the preferential treatment mentioned
         // above).
-        let min_elapsed = PersistConfig::FAKE_READ_LEASE_DURATION / 4;
-        let elapsed_since_last_heartbeat = self.last_heartbeat.elapsed();
+        let min_elapsed = (PersistConfig::FAKE_READ_LEASE_DURATION / 4)
```

Comment:
I'd prefer for us to do this comparison the other way: leave this as a Duration and convert `elapsed` (now computed in ms) to a Duration for the `>=`

also, `self.last_heartbeat` and `elapsed_since_last_heartbeat` probably want `_ms` suffixes for clarity